### DISCUSSION TITLE
What is this problem and how can I solve it please?

### DISCUSSION THREAD

**Discussion by HAMZAWY2212** [lllyasviel/Fooocus#3779] [Q&A] [answered]
![Annotation 2024-12-11 094114](https://github.com/user-attachments/assets/cb7cb28b-19e1-4b7e-8048-bb8bb140fb3a)

**Comment by mashb1t** [comment, marked as answer]
You're missing cv2, please install the package manually or follow the official installation instructions in the readme.
